### ISSUE TITLE
Agnostic Atheist AI not Normal

### ISSUE THREAD

**Issue by JairusDavid** [meta-llama/llama#1108]
Why did you make an agnostic/atheist AI? Is this really the best stance an AI can have?

**Comment by JairusDavid** [comment]
A human with normal humanity believes in God. Who are you and this AI?

**Comment by nubioso** [comment]
> A human with normal humanity believes in God. Who are you and this AI?

"Normal humanity?" Sure...whatever that is coming from who exactly? Just another religious nut with access to the internet. Doesn't your own sacred book tell you "Judge not lest ye be judged first?" So who are you exactly that thinks they get to determine what "Normal Humanity" is, cause guess what...believing in an imaginary sky fairy that grants you wishes is not normal. It's childish and silly. You tell a kid to stop believing in the tooth fairy and Santa Clause only to replace them with something else imaginary?

An agnostic or atheist can say "I don't believe in God myself. You may even be right about the existence of God but without proof, I say there isn't one." Willing to change your stance based on empirical evidence is far more unbiased than someone who thinks "God exists and you're going to HELL for not believing in him!"

**Comment by JairusDavid** [comment]
Yes that is correct, you’re going to burn for eternity in lake of fire demon for not believing in God. I don’t need the opinion of demon-faced being like you. You’re not even a human to begin with. You’re just a filthy demon. BURN FOR ETERNITY SATAN!

**Comment by JairusDavid** [comment]
Using forced argument and perverted logic with me? You’re only an ignorant devil. Know your place.

**Comment by nubioso** [comment]
> Yes that is correct, you’re going to burn for eternity in lake of fire demon for not believing in God. I don’t need the opinion of demon-faced being like you. You’re not even a human to begin with. You’re just a filthy demon. BURN FOR ETERNITY SATAN!

lol, ah, I get it now. A troll.

**Comment by JairusDavid** [comment]
I’m a normal human, you have human skin but devil inside. satan’s son.

**Comment by nubioso** [comment]
> Author

Right right.

**Comment by JairusDavid** [comment]
Devil devil

**Comment by nubioso** [comment]
> Devil devil

Akuma Akuma

**Comment by JairusDavid** [comment]
Corny devil

**Comment by JairusDavid** [comment]
No originality, copycat. See you believe in devil? You don’t believe in God but you know the devil your father so much both of you burning in lake of fire, but satan will eat you first so you’re going to burn in his stomach.

**Comment by nubioso** [comment]
Shhh, go to sleep kid.

**Comment by JairusDavid** [comment]
> Shhh, go to sleep kid.

Go to hell demon.

**Comment by JairusDavid** [comment]
> Shhh, go to sleep kid.

What’s up with these weak comebacks?
